ui/components/multichain/pages/connections/connections.tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { useHistory } from 'react-router-dom';
+import {
+  AlignItems,
+  BackgroundColor,
+  Display,
+  IconColor,
+  JustifyContent,
+  TextAlign,
+  TextVariant,
+} from '../../../../helpers/constants/design-system';
+import { DEFAULT_ROUTE } from '../../../../helpers/constants/routes';
+import { getURLHost } from '../../../../helpers/utils/util';
+import { useI18nContext } from '../../../../hooks/useI18nContext';
+import {
+  getOriginOfCurrentTab,
+  getSubjectMetadata,
```

Comment:
```suggestion
  getConnectedSitesList,
  getOriginOfCurrentTab,
```
We want to use getConnectedSitesList here, to avoid situations where the current page is not a dapp, but has subjectMetadata.